packages/next/cli/next-dev.ts

```diff
@@ -260,7 +260,7 @@ If you cannot make the changes above, but still want to try out\nNext.js v13 wit
       }
       console.log(feedbackMessage)
 
-      loadBindings()
+      return loadBindings()
         .then((bindings: any) => {
```

Comment:
Why are we wrapping in a async function with a promise constructor? Can we just use the `await` syntax and wrap in a `try`/`catch`?

Reply:
it's mostly leftover from last-minute error message changes and we may refactor this logic bit later, including fixing some handlings in error message part itself.